src/couch_mrview/src/couch_mrview_debug.erl

```diff
@@ -48,3 +78,36 @@ view_signature(DbName, DDocName) ->
     {ok, DDoc} = couch_db:open_doc_int(Db, <<"_design/", DDocName/binary>>, []),
     {ok, IdxState} = couch_mrview_util:ddoc_to_mrst(DDocName, DDoc),
     couch_util:to_hex(IdxState#mrst.sig).
+
+view_state(Pid) ->
+    {ok, IdxState} = couch_index:get_state(Pid, 0),
+    case IdxState of
+        #mrst{} ->
+            {ok, Info} = couch_index:get_info(Pid),
+            Sig = IdxState#mrst.sig,
+            DbName = IdxState#mrst.db_name,
+            MrView = lists:keyfind(mrview, 1, IdxState#mrst.views),
+            {pending_updates, PendingUpdates} = lists:keyfind(pending_updates, 1, Info),
```

Comment:
Should the following pattern match with `{value, {Key, Value}}` that `couch_util:get_value/3` uses?
```
{value,{mrview,0,3,0,
               [<<"dbv1view">>],
               [],<<"function(doc){emit(doc.id, 1);}">>,
               {btree,<7954.955.0>,
                      {109,{2,[],24},51},
                      undefined,undefined,
                      fun couch_ejson_compare:less_json_ids/2,
                      #Fun<couch_mrview_util.14.69111627>,snappy},
```
I would think not since the parities of the tuples do not match.